logstash-core-event-java/src/test/java/com/logstash/EventTest.java

```diff
@@ -4,6 +4,9 @@
 
 import java.io.IOException;
 import java.util.*;
+
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.core.IsSame.sameInstance;
```

Comment:
looks like it has already been included. I just autocompleted. unless I am mistaken?